docs/data/xrootd/install-standalone.md

```diff
@@ -1,2 +1,182 @@
 Install XRootD Gateway
 ======================
+
+[XRootD](http://xrootd.org/) is a hierarchical storage system that can be used in a variety of ways to access data,
+typically distributed among actual storage resources. In this document we focus on using XRootD as a simple layer
+exporting an underlying storage system (e.g., [HDFS](/data/install-hadoop.md)) to the outside world.
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following points:
+
+-   **User IDs:** If it does not exist already, the installation will create the Linux user ID `xrootd`
+-   **Service certificate:** The XRootD service uses a host certificate and key pair at
+    `/etc/grid-security/xrd/xrdcert.pem` and `/etc/grid-security/xrd/xrdkey.pem`
+-   **Networking:** The XRootD service uses port 1094 by default
+
+As with all OSG software installations, there are some one-time (per host) steps to prepare in advance:
+
+-   Ensure the host has [a supported operating system](/release/supported_platforms)
+-   Obtain root access to the host
+-   Prepare [the required Yum repositories](/common/yum)
+-   Install [CA certificates](/common/ca)
+
+Installing XRootD
+-----------------
+
+To install XRootD, run the following Yum command:
+
+``` console
+root@xrootd-server # yum install xrootd
+```
+
+Configuring XRootD
+------------------
+
+You will need to modify the file `/etc/xrootd/xrootd-server.cfg` that we will refer now on (unless explicitly said) as THE configuration file.
```

Comment:
This configuration example needs to be clarified so that admins know what configuration needs to be customized to their site. It looks like there's two things that we're suggesting an admin modify:

- `all.export`: instead of just setting this to `/store/ global`, let's explicitly tell them that they should set this to the directory that users will interact with for storage. Also, what does `global` mean? Do they actually want to edit that? What are the options for editing it? Can we link to the documentation so if admins have questions, they have something they can more easily consult?
- `all.sitename`: How is this used for monitoring? Does it have to match anything like the topology resource name?